x-pack/legacy/plugins/siem/public/components/timeline/footer/index.test.tsx

```diff
@@ -93,38 +93,35 @@ describe('Footer Timeline Component', () => {
     });
 
     test('it renders the Loading... in the more load button when fetching new data', () => {
-      const wrapper = mount(
-        <TestProviders>
-          <Footer
-            serverSideEventCount={mockData.Events.totalCount}
-            hasNextPage={getOr(false, 'hasNextPage', mockData.Events.pageInfo)!}
-            height={100}
-            isLive={false}
-            isLoading={true}
-            itemsCount={mockData.Events.edges.length}
-            itemsPerPage={2}
-            itemsPerPageOptions={[1, 5, 10, 20]}
-            onChangeItemsPerPage={onChangeItemsPerPage}
-            onLoadMore={loadMore}
-            nextCursor={getOr(null, 'endCursor.value', mockData.Events.pageInfo)!}
-            tieBreaker={getOr(null, 'endCursor.tiebreaker', mockData.Events.pageInfo)}
-            getUpdatedAt={getUpdatedAt}
-            compact={compact}
-          />
-        </TestProviders>
+      const wrapper = shallow(
+        <PagingControl
+          hasNextPage={getOr(false, 'hasNextPage', mockData.Events.pageInfo)!}
+          loadMore={loadMore}
+          isLoading={true}
+        />
       );
-      wrapper
-        .find(Footer)
-        .instance()
-        .setState({ paginationLoading: true });
-      wrapper.update();
-      expect(wrapper.find('[data-test-subj="LoadingPanelTimeline"]').exists()).toBeFalsy();
```

Comment:
We are not testing for that anymore, by only rendering the `PagingControl` component

Reply:
I think that all the test should be the same to make sure that our refactoring is still acting the same way

Reply:
whoops, ill add that back in